arviz/data/io_cmdstanpy.py

```diff
@@ -755,6 +794,9 @@ def from_cmdstanpy(
     save_warmup : bool
         Save warmup iterations into InferenceData object, if found in the input files.
         If not defined, use default defined by the rcParams.
+    dtypes: dict or str
```

Comment:
not sure how to combine with this, but I think in general it will be more convenient to pass a `CmdStanModel` instance to the converter and have it take the model code from there, instead of either dict or string. Maybe there is even more useful info to be retrieved from the model object? @mitzimorris?

Reply:
I am quite sure it's in the model, but not in the CmdStanMCMC object that is the only one currently being passed to the converter

Reply:
Program code is read once, but it is not saved.

https://github.com/stan-dev/cmdstanpy/blob/ecd844e9945ecf153be21ab69112d71862d2e0be/cmdstanpy/model.py#L130

I don't think there is anything needed from the CmdStanModel object (at least for now).

Reply:
Ok, there is `.code` method, but I think if users need to give model object, then calling `.code` might do the same thing.


Should we add option to accept CmdStanModel?

Reply:
Updated